ci-scripts/copy_to_archive

```diff
@@ -30,22 +30,30 @@ SDK_DIR="${BUILD_DIR}/tmp/deploy/sdk"
 
 RPI_ARCH="raspberrypi3"
 INTEL_ARCH="intel-corei7-64"
+ARP_ARCH="arp"
 
 IMAGE_BASENAME="core-image-pelux"
 SDK_BASENAME="pelux-glibc"
 
+##### RPI image storing is disabled for now to save capacity #####
 # If there is a rpi directory available
-if [ -d ${IMAGE_DIR}/${RPI_ARCH} ]; then
-    echo "Archiving rPi image to $ARCHIVE_DIR"
-    cp ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*.rpi-sdimg $ARCHIVE_DIR
-fi
+#if [ -d ${IMAGE_DIR}/${RPI_ARCH} ]; then
```

Comment:
why leaving this commented?

Reply:
In case someone wanted to enable it easily, but I guess that could also be done using the git history.